docs/whatsnew/4.0.rst

```diff
@@ -1,5 +1,3 @@
-.. doctest-skip-all
```

Comment:
Should we not skip this file?

Reply:
Why? It seems like a fine idea to check that the output is as promised. Perhaps we want to skip it *after* 4.0 is released, in case future changes affect the output but we still want the output to reflect what it would have done in 4.0.

Reply:
These files have a tendency to break with time, so we normally skip them now.